bulk-ingester/src/main/java/org/alfresco/hxi_connector/bulk_ingester/processor/mapper/AlfrescoNodeMapper.java

```diff
@@ -62,21 +64,20 @@
 @Slf4j
 @Component
 @RequiredArgsConstructor
-public class AlfrescoNodeMapper
-{
+public class AlfrescoNodeMapper {
```

Comment:
Please can you use the formatter provided? You can use pre-commit to ensure you format everything automatically.